packages/sdks/src/blocks/columns/columns.lite.tsx

```diff
@@ -206,6 +207,23 @@ export default function Columns(props: ColumnProps) {
     },
   });
 
+  onInit(() => {
+    useTarget({
+      angular: () => {
+        state.flexDir =
+          props.stackColumnsAt === 'never'
+            ? 'row'
+            : props.reverseColumnsWhenStacked
+              ? 'column-reverse'
+              : 'column';
+        state.gutterSize =
+          typeof props.space === 'number' ? props.space || 0 : 20;
+        state.cols = props.columns || [];
+        state.stackAt = props.stackColumnsAt || 'tablet';
+      },
+    });
+  });
```

Comment:
I see you've added many `onInit` hooks that are duplicates of the initial values for certain `state.*` values.

I think we should improve the mitosis generator so that it does this automatically: any initial `state` value should be initialized properly without need for duplication